### PR TITLE
Release notes for R3 release

### PR DESCRIPTION
**Pull request by gvbalaji**
Closes: https://github.com/nephio-project/nephio/issues/769

### CODE REVIEW

File: content/en/docs/release-notes/R3.md [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+
+### Packages
+
+* Catalog fixes
+* Packages to deploy OSC RIC
+
+### Functionalities
+
+* Platform
+  * Fixes, Security patches and enhancements to Porch. See [Appendix](#Appendix) for details.
```
Comment (radoslawc) [inline content/en/docs/release-notes/R3.md:45]:
```suggestion
  * Fixes, Security patches and enhancements to Porch. See [Appendix](#appendix) for details.
```

**Comment by CsatariGergely** [comment]
Fixes: https://github.com/nephio-project/nephio/issues/769

**Review by liamfallon — COMMENTED** [review]
/approve
/lgtm

**Pull request merged**